WebXR API.md

```diff
@@ -228,6 +272,10 @@ _The XRPresentationFrame differs from the VRPresentationFrame with the addition
 - Should we fire an event when a marker or feature based anchor (e.g. a wall, a table top) is detected?
 - How can we access camera image buffers aor textures?
 
+`getCameraFrame` allows developers to access camera image and pose data. For some devices, the camera frame is not always available in each presentation frame. `getCameraFrame` returns `XRCameraFrame` object when sychronized camera frame is available to current presentation frame, otherwise, it returns nothing. 
+
+When `XRCameraFrame` available, developers are able to draw the camera image by WebGL insdie the rendering loop for devices. Or developers are able to send the camera frame to a worker thread for computer vision processing, e.g. marker detection, in parallel.
+
```

Comment:
> Another question: does providing the data inside the frame means the CV happens inside the rAF callback? 

@blairmacintyre , the CV happens on worker thread. If developers want to render the camera image, it happens inside the rAF callback.